frontend/public/static/locale/en-US/translate.ftl

```diff
@@ -1,18 +1,11 @@
 ### Localization for the Translate page of Pontoon
 
-
-## Editor
-## Allows contributors to modify or propose a translation
-
-editor-editor-sign-in-to-translate = <a>Sign in</a> to translate.
-editor-editor-read-only-localization = This is a read-only localization.
-editor-editor-button-copy = Copy
-editor-editor-button-clear = Clear
-editor-editor-button-save = Save
-editor-editor-button-suggest = Suggest
+# Naming convention for l10n IDs: "module-ComponentName--sentence-summary".
+# This allows us to minimize the risk of conflicting IDs throughout the app.
+# Please sort alphabetically by (module name, component name).
```

Comment:
Some items in the `editor` and `entitydetails` modules are not sorted by component (`EditorMenu` and `EditorSettings` should come before `FailedChecks`, `EntityNavigation` should also be moved up).